dll/cpl/powercfg/lang/ro-RO.rc

```diff
@@ -117,64 +117,64 @@ CAPTION "Hibernare"
-    DEFPUSHBUTTON "Confirmă", IDOK, 79, 40, 50, 14
-    PUSHBUTTON "Anulează", IDCANCEL, 133, 40, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 79, 40, 50, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 133, 40, 50, 14
 END
 
 STRINGTABLE
 BEGIN
-    IDS_CPLNAME_1 "Alimentare"
-    IDS_CPLDESCRIPTION_1 "Configurarea metodelor de economisire a consumului de energie al calculatorului."
+    IDS_CPLNAME_1 "Opţiuni de alimentare"
+    IDS_CPLDESCRIPTION_1 "Configurează setările de economisire a energiei pentru computer."
     IDS_PERCENT "%i%%"
     IDS_SOUND "sonoră"
     IDS_TEXT "text"
-    IDS_CONFIG1 "Configurare regim %s"
-    IDS_CONFIG2 "Opțiuni avansate pentru %s"
-    IDS_NOACTION "nespecificată"
-    IDS_PowerActionNone1 "acțiune nespecificată"
-    IDS_PowerActionUnknown "acțiune necunoscută"
-    IDS_PowerActionSleep "intrarea în stare de veghe"
-    IDS_PowerActionHibernate "intrarea în hibernare"
-    IDS_PowerActionShutdown "închiderea calculatorului"
+    IDS_CONFIG1 "Setări pentru schema de alimentare %s"
+    IDS_CONFIG2 "Setări complexe pentru %s"
+    IDS_NOACTION "Fără"
+    IDS_PowerActionNone1 "Nici o acţiune"
+    IDS_PowerActionUnknown "Necunoscut"
+    IDS_PowerActionSleep "Stare de veghe"
+    IDS_PowerActionHibernate "Hibernare"
+    IDS_PowerActionShutdown "Închidere"
     IDS_PowerActionRestart "repornirea calculatorului"
     IDS_PowerActionShutdownOff "închiderea și deconectarea de la alimentare"
     IDS_PowerActionWarmEject "solicitarea acțiunii utilizatorului"
-    IDS_PowerActionNone2 "nespecificat"
-    IDS_TIMEOUT1 "după 1 min"
-    IDS_TIMEOUT2 "după 2 min"
-    IDS_TIMEOUT3 "după 3 min"
-    IDS_TIMEOUT4 "după 5 min"
-    IDS_TIMEOUT5 "după 10 min"
-    IDS_TIMEOUT6 "după 15 min"
-    IDS_TIMEOUT7 "după 20 min"
-    IDS_TIMEOUT8 "după 25 min"
-    IDS_TIMEOUT9 "după 30 min"
-    IDS_TIMEOUT10 "după 45 min"
-    IDS_TIMEOUT11 "după 1 oră"
-    IDS_TIMEOUT12 "după 2 ore"
-    IDS_TIMEOUT13 "după 3 ore"
-    IDS_TIMEOUT14 "după 4 ore"
-    IDS_TIMEOUT15 "după 5 ore"
-    IDS_TIMEOUT16 "dezactivată"
-    IDS_DEL_SCHEME "Sigur doriți eliminarea aceastui regim de consum energetic: ""%s""?"
-    IDS_DEL_SCHEME_TITLE "Eliminarea regumului de consum"
-    IDS_DEL_SCHEME_ERROR "Nu poate fi ștearsă schema de alimentare"
+    IDS_PowerActionNone2 "Fără"
+    IDS_TIMEOUT1 "După 1 minut"
+    IDS_TIMEOUT2 "După 2 minute"
+    IDS_TIMEOUT3 "După 3 minute"
+    IDS_TIMEOUT4 "După 5 minute"
+    IDS_TIMEOUT5 "După 10 minute"
+    IDS_TIMEOUT6 "După 15 minute"
+    IDS_TIMEOUT7 "După 20 minute"
+    IDS_TIMEOUT8 "După 25 minute"
+    IDS_TIMEOUT9 "După 30 minute"
+    IDS_TIMEOUT10 "După 45 minute"
+    IDS_TIMEOUT11 "După 1 oră"
+    IDS_TIMEOUT12 "După 2 ore"
+    IDS_TIMEOUT13 "După 3 ore"
+    IDS_TIMEOUT14 "După 4 ore"
+    IDS_TIMEOUT15 "După 5 ore"
+    IDS_TIMEOUT16 "Niciodată"
+    IDS_DEL_SCHEME "Sigur ştergeţi schema de alimentare: ""%s""?"
```

Comment:
```suggestion
    IDS_DEL_SCHEME "Sigur ştergeţi schema de alimentare: '%s'?"
```
Just an idea. I am not sure what MS does use here actually. No need to change it if you prefer the other way.

Reply:
I was tired yesterday and I didn't pay attention. In every language there is "" sign and not ''. This is the same in Windows, too. Here is a screenshot. I will revert this change.

![2024-04-14 11_29_29-](https://github.com/reactos/reactos/assets/113553683/7d33017d-99a4-4764-8fe5-deaf26f7b8c6)

Reply:
Ok, no problem. Resolved, thanks.